readme.md

```diff
@@ -60,6 +60,29 @@ Devices and data sources in an IoT solution can range from a simple network-conn
 
+### Provisioning client SDK
+:heavy_check_mark: feature available  :heavy_multiplication_x: feature planned but not supported  :heavy_minus_sign: no support planned
+This repository contains [provisioning device client SDK](./provisioning-device-client) for the [Device Provisioning Service](https://docs.microsoft.com/en-us/azure/iot-dps/). 
+
+| Features                    | mqtt               | mqtt-ws            | amqp               | amqp-ws            | https              | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+|-----------------------------|--------------------|--------------------|--------------------|--------------------|--------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| TPM Individual Enrollment   | :heavy_check_mark: | :heavy_multiplication_x: | :heavy_check_mark: | :heavy_multiplication_x: | :heavy_check_mark: | This SDK supports connecting your device to the Device Provisioning Service via [individual enrollment](https://docs.microsoft.com/en-us/azure/iot-dps/concepts-service#enrollment) using [Trusted Platform Module](https://docs.microsoft.com/en-us/azure/iot-dps/concepts-security#trusted-platform-module-tpm).  This [quickstart](https://docs.microsoft.com/en-us/azure/iot-dps/quick-create-simulated-device) reviews how to create a simulated device for individual enrollment with TPM. Websocket connection is currently not available.                                                                                                                                                                                                               |
```

Comment:
Actually TPM over MQTT is not supported. Can you change that ?

Reply:
This is true for all SDK's. MQTT only has X509 support.

Reply:
You also specififcally say AMQP_WS is work in progress.

Reply:
TPM over MQTT is not supported on all SDKs?  @tameraw can you confirm that?